Scripts/build.swift

```diff
@@ -20,38 +20,132 @@ enum TaskError: Error {
     case code(Int32)
 }
 
-guard CommandLine.arguments.count > 3 else {
-    print("Usage: build.swift sdk destination should_test? enable_code_coverage?")
+enum Platform: String, CaseIterable, CustomStringConvertible {
+    case iOS_12
+    case iOS_13
+    case tvOS_12
+    case tvOS_13
+    case macOS_10_15
```

Comment:
Note that we don't have a `macOS_10_14`, since this project requires Xcode 11 to build, and there is no CI configuration on Travis CI that uses macOS 10.14's SDK but enables building with Xcode 11.